suplemon/modules/keymap.py

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8
+
+import os
+
+from suplemon.suplemon_module import Module
+
+
+class Keymap(Module):
```

Comment:
I'm pretty sure we could make this `KeymapConfig` which inherits from a `Config` class and rename the existing `Config` to `SuplemonConfig` and also inherit from `Config`